client/src/routes/DiscoverPage.tsx

```diff
@@ -21,6 +21,7 @@ const StyledSearchContainer = styled.View`
 `
 
 const DiscoverPage = () => {
+  const [recipes, setRecipes] = useState<RecipeObject[]>()
```

Comment:
tää on tähä hätää viel fine, poistuupi heti reduxin kohalla

Reply:
Joo näitä just meinasin et saa sit refaktoroituu suoraa reduxii ku tulee se tost toiselt pullarilt